docs/developers.md

```diff
@@ -0,0 +1,39 @@
+# Linting
+
+To ensure consistent formatting across developres, we include congifurations for Prettier (code formatting), ESLint (JS), and StyleLint (SCSS) to apply consistent styles.
+
+Inside of the `website` folder, you can run `yarn eslint` to list issues, or `yarn eslint:fix` to automatically fix what is possible.
+
+Similarly to lint stylesheets, you can run `yarn stylelint` or `yarn stylelint:fix`.
+
+To lint as you go, we reccomend configuring your editor to use this config and apply fixes on save, and display warnings. For VSCode, you can do that by installing the eslint, prettier and stylelint plugins.
+
+To ensure we're using our project specified configuration, you'll want to open settings via `Preferences: Open Settings (JSON)`, and add the following rules:
+
+```json
```

Comment:
So far we've always excluded `.vscode` as a whole. Do you think we should still do that but allow this one file though?

Reply:
i'm not sure what else lands in the `.vscode` folder. it seems very workspace-specific. from FAQ:

> ## When does it make sense to use workspace settings?
> If you're using a workspace that needs custom settings but you don't want to apply them to your other VS Code projects. A good example is language-specific linting rules.

This makes me think the .vscode folder was specifically designed for what we're trying to do here. It's pretty opinionated but I think that's the goal.

As some users (like me) might have `.vscode` globally gitignored, it might be helpful to use [this take on a gitignore](https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore) of `.gitignore`:

Reply:
Main Next repo uses it, which is good enough for me.
https://github.com/vercel/next.js/blob/canary/.vscode/settings.json

 Going go add here.